Tests/ApiEndpoints/test_allhousingapp_pytest.py

```diff
@@ -121,6 +182,44 @@ def test_get_application_id_not_found(self):
         if not response.status_code == 404:
             pytest.fail('Expected 404 Not Found, got {0}.'\
                 .format(response.status_code))
+
+#    Verify that an application's date submitted is changed successfully
+#    Endpoint -- 'api/housing/apartment/applications/{applicationID}/submit'
+#    Expected Status Code -- 200 OK
+#    Expected Content -- a non-null value for dateSubmitted 
+    def test_date_submitted_changed(self):
+        self.session = self.createAuthorizedSession(control.username, control.password)
+        self.url = control.hostURL + 'api/housing/apartment/applications'
+        self.data = {
+            'ApplicationID' : -1,
+            'EditorUsername' : control.leader_username,
+            'Applicants' : [
+                {
+                    'Username' : control.leader_username,
+                },
+            ],
+            'ApartmentChoices' : [
+                {
+                    'HallRank' : 1,
+                    'HallName' : "Taj MaHall"
```

Comment:
Same here

Reply:
No problem. I will change them to real halls